redmine/issue.rb

```diff
@@ -19,6 +20,18 @@ def subject
     @raw_data['issue']['subject']
   end
 
+  def backlog?
+    version_name[/backlog/i]
```

Comment:
Will this work? `version_name` can be nil in which case I think this will error out. Did you mean `version_name =~ /backlog/i`?